Android.gitignore

```diff
@@ -41,11 +22,7 @@ captures/
 .idea/libraries
 
 # Keystore files
-# Uncomment the following line if you do not want to check your keystore files in.
-#*.jks
-
-# External native build folder generated in Android Studio 2.2 and later
-.externalNativeBuild
+*.jks
```

Comment:
When you create a new keystore, Android Studio point you to create a *.jks file, however at the end of the process it creates a file without extension. Actually it's a developer job to rename the file adding the extension which is not needed to let the file works correctly.

That said, developers can add .jks or .keystore extension to the file, and it will work in both cases. When Android first came out Eclipse created .keystore files and if we want cover both the cases i think you should add *.keystore too to this line

Reply:
Good point. I'm not sure about the keystore generation on Windows, but the official guidelines to [generate the file on command line](https://developer.android.com/studio/publish/app-signing.html#signing-manually) use `my-release-key.jks` as the sample filename, so `*.jks` would match it.
I'm open to also add `*.keystore` if others think it would be useful. I have not seen such a file in a long time, but do remember that the file used to have this extension.

Reply:
Yeah there is a problem about this, using the CLI you can specify the extension while using the GUI if you dont manually add ".jks" at the end of the name it is not added automatically (even if the suggestion say it should be).

There is the potential to open an issue in the Android tracker about this "strange" behaviour

Reply:
Okay sure, I'll add `*.keystore` as well.